lib/o365_mgmnt/index.js

```diff
@@ -46,13 +33,13 @@ var Office365Env = {
     azureDataLakeAnalyticsCatalogAndJobEndpointSuffix: 'azuredatalakeanalytics.net'
 };
 
-var g_appAdCreds = new msRestAzure.ApplicationTokenCredentials(
-        process.env.CUSTOMCONNSTR_APP_CLIENT_ID,
-        process.env.APP_TENANT_ID,
-        process.env.CUSTOMCONNSTR_APP_CLIENT_SECRET,
-        { 'tokenCache': tokenCache,
-          environment : Office365Env 
-        });
+var g_appAdCreds = new ApplicationTokenCredentials(
+    process.env.CUSTOMCONNSTR_APP_CLIENT_ID,
+    process.env.APP_TENANT_ID,
+    process.env.CUSTOMCONNSTR_APP_CLIENT_SECRET,
+    null,
```

Comment:
Actually looks like we can now use token audience here, please see https://github.com/Azure/ms-rest-nodeauth/blob/master/lib/credentials/applicationTokenCredentials.ts#L20. And we can get rid of this `Office365Env `

Could you please try to use `https://manage.office.com` audience and see if o365 management api requests are successful?

Reply:
Good spot. Definielty pays to scroll over and read the who comment!